int64/int64.mbt

```diff
@@ -47,25 +47,3 @@ pub fn Int64::max_value() -> Int64 {
 pub fn Int64::min_value() -> Int64 {
   min_val
 }
-
-pub fn hash(self : Int64) -> Int {
-  let b = Bytes::make(8)
-  b[0] = self.lsr(56).to_byte()
-  b[1] = self.lsr(48).land(0xFFL).to_byte()
-  b[2] = self.lsr(40).land(0xFFL).to_byte()
-  b[3] = self.lsr(32).land(0xFFL).to_byte()
-  b[4] = self.lsr(24).land(0xFFL).to_byte()
-  b[5] = self.lsr(16).land(0xFFL).to_byte()
-  b[6] = self.lsr(8).land(0xFFL).to_byte()
-  b[7] = self.land(0xFFL).to_byte()
-  b.hash()
```

Comment:
`Bytes::[a,b,c]`

Reply:
I think the current implementation is more intuitive so I left it as is.